### PR TITLE
Release/1.6.3

### PR DESCRIPTION
**Pull request by AndyTWF**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,29 @@
...
+## [1.6.3](https://github.com/ably/ably-asset-tracking-android/tree/v1.6.3)
+
+[Full Changelog](https://github.com/ably/ably-asset-tracking-android/compare/v1.6.2...v1.6.3)
+
+**Fixed bugs:**
+
+- Re-adding a trackable results in trackable being offline [\#989](https://github.com/ably/ably-asset-tracking-android/issues/989)
+- Re-added trackable does not enter or subscribe to presenc [\#988](https://github.com/ably/ably-asset-tracking-android/issues/988)
+- Re-adding trackable immediately after removing it causes NullPointerException [\#984](https://github.com/ably/ably-asset-tracking-android/issues/984)
+- Exceptions thrown inside a block passed to `DefaultAbly`’s `scope.launch()` are suppressed [\#909](https://github.com/ably/ably-asset-tracking-android/issues/909)
```
Comment (ikbalkaya) [inline CHANGELOG.md:12]: This looks like an internal change 🤔 I'd personally remove it from here if it was.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,29 @@
...
+
+- Remove `ConnectionException` from docstrings of `Publisher.add/track` [\#995](https://github.com/ably/ably-asset-tracking-android/issues/995)
+- Improve the version `1.5.1` to `1.6.0` migration guide [\#974](https://github.com/ably/ably-asset-tracking-android/issues/974)
+- Remove workaround for handling failed connection when adding a trackable [\#898](https://github.com/ably/ably-asset-tracking-android/issues/898)
+
+**Merged pull requests:**
+
+- Remove `ConnectionException` from docstrings of `Publisher.add/track` [\#997](https://github.com/ably/ably-asset-tracking-android/pull/997) ([lawrence-forooghian](https://github.com/lawrence-forooghian))
+- Add migration guides for 1.6.0 and 1.6.1 [\#996](https://github.com/ably/ably-asset-tracking-android/pull/996) ([lawrence-forooghian](https://github.com/lawrence-forooghian))
+- Test exceptions [\#994](https://github.com/ably/ably-asset-tracking-android/pull/994) ([davyskiba](https://github.com/davyskiba))
```
Comment (ikbalkaya) [inline CHANGELOG.md:24]: This also looks like an internal change

**Review by ikbalkaya — APPROVED** [review]
I left some comments on some items that could be removed from Changelog. I'll leave it to you whether you want to apply those.

File: CHANGELOG.md (hunk 1 above)
Comment (AndyTWF) [reply CHANGELOG.md:12]: It looks like historically we've kept internal changes in the changelog - for example in 1.5.0?

**Review by davyskiba — APPROVED** [review]

**Pull request merged**